controllers/dscinitialization/servicemesh_setup.go

```diff
@@ -151,7 +154,7 @@ func (r *DSCInitializationReconciler) serviceMeshCapabilityFeatures(instance *ds
 
 		cfgMapErr := feature.CreateFeature("mesh-shared-configmap").
 			For(handler).
-			WithResources(servicemesh.MeshRefs, servicemesh.AuthRefs).
+			WithResources(servicemesh.MeshRefs, servicemesh.AuthRefs(GetEffectiveClusterAudiences(handler.ServiceMesh.Auth.Audiences, r.Client, r.Log))).
```

Comment:
I think if we extract it to variable it will be a bit more readable, considering its length now.